src/Diagnostics.ModelsAndUtils/Models/Resource/App.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Diagnostics.ModelsAndUtils.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Models
+{
+    /// <summary>
+    /// Resource representing an app service
+    /// </summary>
+    public class App : AppFilter, IResource
```

Comment:
So potentially, since App is inheriting from AppFilter, App class can be used as an attribute right?